src/app/helptext/sharing/webdav/webdav.ts

```diff
@@ -24,5 +24,11 @@ export const helptext_sharing_webdav = {
     column_comment: T('Comment'),
     column_path: T('Path'),
     column_ro: T('Read Only'),
-    column_perm: T('Change User & Group Ownership')
+    column_perm: T('Change User & Group Ownership'),
```

Comment:
Only use "&" in the name of a law firm.
```suggestion
    column_perm: T('Change User and Group Ownership'),
```